bundles/org.openhab.binding.volvooncall/src/main/java/org/openhab/binding/volvooncall/internal/handler/VehicleHandler.java

```diff
@@ -13,9 +13,69 @@
+import static org.openhab.binding.volvooncall.internal.VolvoOnCallBindingConstants.LOCATION_TIMESTAMP;
+import static org.openhab.binding.volvooncall.internal.VolvoOnCallBindingConstants.LOCK;
+import static org.openhab.binding.volvooncall.internal.VolvoOnCallBindingConstants.ODOMETER;
+import static org.openhab.binding.volvooncall.internal.VolvoOnCallBindingConstants.PRECLIMATIZATION;
+import static org.openhab.binding.volvooncall.internal.VolvoOnCallBindingConstants.REAR_LEFT;
+import static org.openhab.binding.volvooncall.internal.VolvoOnCallBindingConstants.REAR_LEFT_TYRE;
+import static org.openhab.binding.volvooncall.internal.VolvoOnCallBindingConstants.REAR_LEFT_WND;
+import static org.openhab.binding.volvooncall.internal.VolvoOnCallBindingConstants.REAR_RIGHT;
+import static org.openhab.binding.volvooncall.internal.VolvoOnCallBindingConstants.REAR_RIGHT_TYRE;
+import static org.openhab.binding.volvooncall.internal.VolvoOnCallBindingConstants.REAR_RIGHT_WND;
+import static org.openhab.binding.volvooncall.internal.VolvoOnCallBindingConstants.REMOTE_HEATER;
+import static org.openhab.binding.volvooncall.internal.VolvoOnCallBindingConstants.SERVICE_URL;
+import static org.openhab.binding.volvooncall.internal.VolvoOnCallBindingConstants.SERVICE_WARNING;
+import static org.openhab.binding.volvooncall.internal.VolvoOnCallBindingConstants.TAILGATE;
+import static org.openhab.binding.volvooncall.internal.VolvoOnCallBindingConstants.TIME_TO_BATTERY_FULLY_CHARGED;
+import static org.openhab.binding.volvooncall.internal.VolvoOnCallBindingConstants.TRIPMETER1;
+import static org.openhab.binding.volvooncall.internal.VolvoOnCallBindingConstants.TRIPMETER2;
+import static org.openhab.binding.volvooncall.internal.VolvoOnCallBindingConstants.TRIP_CONSUMPTION;
+import static org.openhab.binding.volvooncall.internal.VolvoOnCallBindingConstants.TRIP_DISTANCE;
+import static org.openhab.binding.volvooncall.internal.VolvoOnCallBindingConstants.TRIP_DURATION;
+import static org.openhab.binding.volvooncall.internal.VolvoOnCallBindingConstants.TRIP_END_POSITION;
+import static org.openhab.binding.volvooncall.internal.VolvoOnCallBindingConstants.TRIP_END_TIME;
+import static org.openhab.binding.volvooncall.internal.VolvoOnCallBindingConstants.TRIP_START_ODOMETER;
+import static org.openhab.binding.volvooncall.internal.VolvoOnCallBindingConstants.TRIP_START_POSITION;
+import static org.openhab.binding.volvooncall.internal.VolvoOnCallBindingConstants.TRIP_START_TIME;
+import static org.openhab.binding.volvooncall.internal.VolvoOnCallBindingConstants.TRIP_STOP_ODOMETER;
+import static org.openhab.binding.volvooncall.internal.VolvoOnCallBindingConstants.UNDEFINED;
+import static org.openhab.binding.volvooncall.internal.VolvoOnCallBindingConstants.UNLOCK;
+import static org.openhab.binding.volvooncall.internal.VolvoOnCallBindingConstants.UNLOCK_TIME;
+import static org.openhab.binding.volvooncall.internal.VolvoOnCallBindingConstants.WASHER_FLUID_LEVEL;
```

Comment:
Wasn't ```import static org.openhab.binding.volvooncall.internal.VolvoOnCallBindingConstants.*;``` more readable ?

Reply:
It's a setting in my IntelliJ based on the preferences from the client project in my day job.

If a * import is the preferred way in OH I can change that setting and do a reformat

Reply:
Yes, I do not remember seeing such import examples in OH